myapp/spec/system/labels_spec.rb

```diff
@@ -0,0 +1,376 @@
+        let(:input_values) { { name: '1' * 65 } }
+
+        it 'データ登録されていないこと' do
+          visit new_label_path
+          fill_in 'label[name]', with: input_values[:name]
+          click_on '作成'
+          expect(Label.find_by(name: input_values[:name])).to be_nil
+        end
+
+        it 'エラーメッセージが表示されること' do
+          visit new_label_path
+          fill_in 'label[name]', with: input_values[:name]
+          click_on '作成'
+          expect(page).to have_content 'ラベル名は64文字以内で入力してください'
+        end
+      end
+    end
+
+    describe '入力エリア' do
+      let(:input_values) { { name: '新規ラベル' } }
+
+      it '入力した値でラベルが作成されていること' do
+        visit new_label_path
+        fill_in 'label[name]', with: input_values[:name]
+        click_on '作成'
+        expect(Label.find_by(input_values)).to be_present
+      end
+
+      it '作成ボタン押下でラベル一覧画面へ遷移すること' do
+        visit new_label_path
+        fill_in 'label[name]', with: input_values[:name]
+        click_on '作成'
+        expect(page).to have_current_path labels_path
+      end
+
+      it '作成後メッセージが表示されること' do
+        visit new_label_path
+        fill_in 'label[name]', with: input_values[:name]
+        click_on '作成'
+        expect(page).to have_content 'ラベル作成成功'
+      end
+    end
+
+    describe 'フッターエリア' do
+      it 'ラベル一覧へボタン押下でラベル一覧画面へ遷移すること' do
+        visit new_label_path
+        click_on 'ラベル一覧へ'
+        expect(page).to have_current_path labels_path
+      end
+    end
+  end
+
+  describe '#show' do
+    let!(:label) { FactoryBot.create(:label, name: 'label') }
+
+    describe '表示エリア' do
+      it 'ラベル名が一致すること' do
+        visit label_path(label)
+        expect(page).to have_content 'label'
+      end
+    end
+
+    describe 'フッターエリア' do
+      it '編集ボタン押下で編集画面へ遷移すること' do
+        visit label_path(label)
+        click_on '編集'
+        expect(page).to have_current_path edit_label_path(label)
+      end
+
+      it '削除ボタン押下で項目が削除されること' do
+        visit label_path(label)
+        click_on '削除'
+        expect(Label.find_by(id: label.id)).to be_nil
+      end
+
+      it '削除ボタン押下でラベル一覧画面へ遷移すること' do
+        visit label_path(label)
+        click_on '削除'
+        expect(page).to have_current_path labels_path
+      end
+
+      it '削除後メッセージが表示されること' do
+        visit label_path(label)
+        click_on '削除'
+        expect(page).to have_content 'ラベル削除成功'
+      end
+
+      it 'ラベル一覧へボタン押下でタスク一覧画面へ遷移すること' do
+        visit label_path(label)
+        click_on 'ラベル一覧へ'
+        expect(page).to have_current_path labels_path
+      end
+    end
+  end
+
+  describe '#edit' do
+    let!(:label) { FactoryBot.create(:label, name: 'label') }
+
+    describe 'エラー表示エリア' do
+      context '入力エラー（ラベル名未入力）' do
+        let(:input_values) { { name: '' } }
+
+        it 'データが更新されていないこと' do
+          visit edit_label_path(label)
+          fill_in 'label[name]', with: input_values[:name]
+          click_on '更新'
+          expect(Label.find_by(name: input_values[:name])).to be_nil
```

Comment:
全体的に、expectの中でfindしている処理が目立ちします。
letで事前に定義しているはずなので、そのオブジェクトを使うようにしてください。

また、例えばこのテストの場合、画面側では違うオブジェクトを取得し更新しているので、一度letで定義したオブジェクトは最新の情報にする必要があります。
そのときはreloadメソッドが便利です。
```suggestion
          expect(label.reload.name).not_to eq input_values[:name]
```

Reply:
findでなく、reloadを使用するよう全体的に修正しました